main/ertp/guide/purses-and-payments.md

```diff
@@ -1,109 +1,107 @@
+  It can only hold assets of the same `brand` as its source(s).
 
-For any `brand`, any number of `purses` or `payments` can hold assets
-of that `brand`. Neither a `purse` nor a `payment` can ever change their
+Any number of `purses` or `payments` can hold assets
+of any particular `brand`. Neither a `purse` nor a `payment` can ever change their
 associated `brand`.
 
-Each `purse` and `payment` object contains a specific single amount of digital assets, 
-which may be none at all (`empty` in AmountMath terms). In the same way 
-you might have separate bank accounts for different purposes, 
-you can have different purses for the same brand of digital asset. 
-One purse might hold 2 Quatloos and another purse might hold 3 Quatloos. 
+Each `purse` and `payment` object contains a specific single amount of digital assets,
+which may be none at all ("empty" in [AmountMath](./amount-math.md) terms). In the same way
+you might have separate bank accounts for different purposes,
+you can have separate purses for the same `brand` of digital asset.
+One of your purses might hold 2 Quatloos while another holds 9000 Quatloos.
 
-When you deposit assets into a `purse` or `payment`, they are added to
-whatever assets already exist there. So a 3 Quatloos deposit 
-into a `purse` with 8 Quatloos, results in a `purse` with 11 Quatloos. Similarly, 
-then withdrawing 6 Quatloos from that `purse` into a new `payment` leaves the `purse` 
-with 5 Quatloos and the new `payment` has 6 Quatloos. 
+When you deposit assets into a `purse`, they are added to
+whatever assets already exist there. So a 3 Quatloos deposit
+into a `purse` with 8 Quatloos, results in a `purse` with 11 Quatloos. Similarly,
+then withdrawing 6 Quatloos from that `purse` into a new `payment` leaves the `purse`
+with 5 Quatloos and the new `payment` with 6 Quatloos.
 
 When adding a `payment` to a `purse`, you must add the entire `payment`. To
-only add part of a `payment`, you must first use `payment.split()` 
-or `payment.splitMany()` to split the `payment` into two or more new `payments`.
-
-`mints` create entirely new digital assets and put them in a new `payment`. 
-You also create a `payment` by withdrawing assets from a `purse`, by splitting an 
-existing `payment`, or by combining multiple `payments` into one new one. Note 
-the `brand` of the new `payment` is the same as the associated `brand` of its originating `mint`, `purse` or `payment`. 
-
-You don't transfer assets directly from one `purse` to
-another `purse`. Instead, in ERTP you do something like these steps to send and receive assets. The actual send operation is up to you; ERTP does not 
-implement a way of sending object-containing messages between parties.
-- Send assets:
+only add part of a `payment`, you must first use [`issuer.split()`](/ertp/api/issuer.md#issuer-split-payment-paymentamounta)
+or [`issuer.splitMany()`](/ertp/api/issuer.md#issuer-splitmany-payment-amountarray)
+to split it into two or more new `payments`.
```

Comment:
Might as well take this opportunity to deprecate `issuer.split`, `issuer.splitMany`, and `issuer.combine`. In anticipation of https://github.com/Agoric/agoric-sdk/pull/5008 I'd like to discourage `issuer.claim` as well, but perhaps that's premature.

Reply:
Leaving for now while we work through @dtribble's concerns. Deprecation remains easy to add.